<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootWgt</class>
 <widget class="QWidget" name="RootWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#pBtn_search
{
	border-image: url(:/Res/Image/search.png);
	min-width:20;
	max-width:20;
	min-height:20;
	max-height:20;
}

#pEdit_search
{
	padding-left:10;
}

</string>
  </property>
  <widget class="QPushButton" name="pBtn_search">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pEdit_search">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>折子戏</string>
   </property>
   <property name="placeholderText">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>431</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_last">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_next">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>580</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_download">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>链接：</string>
   </property>
  </widget>
  <widget class="QLabel" name="pLbl_status">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pLbl_url">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_setting">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
